### PR TITLE
implement statsmodels handler

### PR DESCRIPTION
**Pull request by isabelizimm**
TODO

- [x] add tests/handling for no ptype predictions
- [x] check edge cases around `statsmodels.base.wrapper.ResultsWrapper` (right now, this seems to work with *most* models that have a `predict` method)

### CODE REVIEW

File: vetiver/handlers/statsmodels.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+class StatsmodelsHandler(BaseHandler):
+    """Handler class for creating VetiverModels with statsmodels.
+
+    Parameters
+    ----------
+    model : statsmodels
+        a trained and fit statsmodels model
+    """
+
+    model_class = staticmethod(lambda: statsmodels.base.wrapper.ResultsWrapper)
```
Comment (isabelizimm) [inline vetiver/handlers/statsmodels.py:22]: for models with a `predict` method, using this class seems to be sufficient to identify `statsmodels` models
